source/features/clean-repo-sidebar.tsx

```diff
@@ -1,17 +1,26 @@
 import './clean-repo-sidebar.css';
+import React from 'dom-chef';
 import select from 'select-dom';
 import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
+import {wrap} from '../helpers/dom-utils';
 
 async function init(): Promise<void> {
 	document.body.classList.add('rgh-clean-repo-sidebar');
 
-	// Hide redundant "Releases" section
+	// Clean up "Releases" section
 	const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]', {waitForChildren: false});
 	if (sidebarReleases) {
-		sidebarReleases.closest<HTMLElement>('h2')!.hidden = true;
+		const releasesSection = sidebarReleases.closest<HTMLElement>('.BorderGrid-row')!;
+		const latestRelease = select('a[href*="/releases/"]', releasesSection);
+		if (latestRelease) {
+			releasesSection.previousElementSibling!.firstElementChild!.append(latestRelease);
+			wrap(latestRelease, <div className="mt-3"/>);
```

Comment:
I’d probably be more comfortable just hiding the header and link, and removing the border and padding. This way we don’t risk breaking the section if they just change some classes (since this code hides the whole thing.

Reply:
That was my intention at first, but I couldn't figure out a way to remove the border between the two sections. The whole sidebar is actually displayed as a table, and I don't think individual cells can disable their own borders.

Reply:
They can, it's plain border and padding. I removed them from the `about` section in the demo screenshot

Reply:
Maybe it's just `about.classList.ass('border-none', 'pb-0')`

Reply:
* `border-none` and `border: none;` have no effect in Firefox. The only way I found to get rid of it is `border-color: transparent;`
 * `py-0` gets rid of the padding on both sides, but that's not exactly what we want